### PR TITLE
pull versions from env variables

### PR DESCRIPTION
**Pull request by remo87**
- read variables from env variables
- fix properties

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -197,6 +197,13 @@ ot {
   }
 }
 
+ot.meta.apiVersion.x=${?META_APIVERSION_MAJOR}
```
Comment (mbdebian) [inline conf/application.conf:200]: Will this use the earlier value as default value in case the corresponding environment variable is not defined?

**Review by mbdebian — COMMENTED** [review]
Thanks for the PR @remo87 , would you mind having a look at the comments?

File: conf/application.conf (hunk 1 above)
Comment (remo87) [reply conf/application.conf:200]: That's correct

**Review by mbdebian — APPROVED** [review]
Thank you @remo87 ! 🥳 
Please, don’t forget to close the associated issue(s)

**Pull request merged**